eth-contracts/test/delegateManager.test.js

```diff
@@ -645,6 +660,58 @@ contract('DelegateManager', async (accounts) => {
         'Expect no lockup funds to carry over')
     })
 
+    it('slash below sp bounds', async () => {
+      let preSlashInfo = await getAccountStakeInfo(stakerAccount, false)
+      // Set slash amount to all but 1 AUD for this SP
+      let diffAmount = toWei(1)
+      let slashAmount = (preSlashInfo.spFactoryStake).sub(diffAmount)
+
+      // Perform slash functions
+      await delegateManager.slash(slashAmount, slasherAccount)
+
+      let isWithinBounds = await serviceProviderFactory.isServiceProviderWithinBounds(slasherAccount)
+      assert.isFalse(
+        isWithinBounds,
+        'Bound violation expected')
+
+      // Initiate round
+      await claimFactory.initiateRound()
+
+      // Confirm claim is pending
+      let pendingClaim = await claimFactory.claimPending(stakerAccount)
+      assert.isTrue(pendingClaim, 'ClaimFactory expected to consider claim pending')
+
+      // Confirm claim fails due to bound violation
+      await _lib.assertRevert(
+        delegateManager.claimRewards({ from: stakerAccount }),
+        'Service provider must be within bounds'
+      )
+
+      // Try to increase by diffAmount, but expect rejection since lower bound is unmet
+      await _lib.assertRevert(
+        increaseRegisteredProviderStake(
+          testEndpoint,
+          diffAmount,
+          stakerAccount),
+        'Minimum stake threshold exceeded')
+
+      // Increase to minimum
+      let bounds = await serviceProviderFactory.getAccountStakeBounds(stakerAccount)
+      let info = await getAccountStakeInfo(stakerAccount, false)
+      let increase = (bounds.min).sub(info.spFactoryStake)
+      // Increase to minimum bound
+      await increaseRegisteredProviderStake(
+        testEndpoint,
+        increase,
+        stakerAccount)
+
+      // Validate increase
+      isWithinBounds = await serviceProviderFactory.isServiceProviderWithinBounds(slasherAccount)
+      assert.isTrue(
+        isWithinBounds,
+        'Valid bound expected')
+    })
```

Comment:
might be good to confirm that claimRewards will now succeed again

Reply:
after our changes, new validation added to confirm that it will actually _not_ succeed again due to stake at fund block